packages/runtime/container-runtime/src/test/pendingStateManager.spec.ts

```diff
@@ -385,7 +414,7 @@ describe("Pending State Manager", () => {
 
 			await pendingStateManager.applyStashedOpsAt(0);
 
-			assert.deepStrictEqual(pendingStateManager.getLocalState().pendingStates, [
+			assert.deepStrictEqual(pendingStateManager.getLocalState()?.pendingStates, [
```

Comment:
Previously `pendingStateManager` was any. Now this code knows `getLocalState()` could be undefined (but it won't be)